modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/opt/GridLuceneFile.java

```diff
@@ -105,6 +116,7 @@ void lockRef() {
     void releaseRef() {
         refCnt.decrementAndGet();
 
+        // TODO: this may trigger actual file deletion but looks like filename would still stay in GridLuceneDirectory.fileMap
```

Comment:
Let's create separate ticket to check that

Reply:
Done